docs/layers.md

```diff
@@ -399,7 +399,7 @@ Places with `kind` values of `continent`, `country`, with others added starting
 #### Place properties (common optional):
 
 * `capital`: a `true` value normalizes values between OpenStreetMap and Natural Earth for kinds of `Admin-0 capital`, `Admin-0 capital alt`, and `Admin-0 region capital`.
-* `state_capital`: a `true` value normalizes values between OpenStreetMap and Natural Earth for kinds of `Admin-1 capital` and `Admin-1 region capital`.
+* `region_capital`: a `true` value normalizes values between OpenStreetMap and Natural Earth for kinds of `Admin-1 capital` and `Admin-1 region capital`.
```

Comment:
Do we also need to update the place kind values below? Seems like some of those are no longer possible, and normalized versions of them are in `kind_detail`.